ocdata/analysis.py

```diff
@@ -0,0 +1,66 @@
+"""
+This submodule contains the scripts for VASP calculations
+post-analysis:
+
+(1) Bader Charges Analysis
+(2) pDOS Analysis (TODO)
+"""
+
+__author__ = 'Javier Heras-Domingo'
+__email__ = 'jherasdo@andrew.cmu.edu'
+
+import os
+import sys
+import pickle
+from shutil import which
+
+import warnings
+warnings.filterwarnings("ignore")
+
+
+
+#Run Bader Analysis after vasp calculation
+def run_bader(outdir='./'):
+	"""
+	Arguments
+	--------------
+	outdir [str]: Directory of the vasp job
```

Comment:
It might be more flexible to use the name of the pickle file as an argument rather than the output directory. This way, users can name the pickle whatever they want. They can also specify where they want to save it by prefixing it to the filename.